exercises/protein-translation/canonical-data.json

```diff
@@ -209,7 +285,78 @@
           "input": {
             "strand": "UGGUGUUAUUAAUGGUUU"
           },
-          "expected": ["Tryptophan","Cysteine","Tyrosine"]
+          "expected": [
+            "Tryptophan",
+            "Cysteine",
+            "Tyrosine"
+          ]
+        },
+        {
+          "uuid": "1e75ea2a-f907-4994-ae5c-118632a1cb0f",
```

Comment:
This is the first time an error case has been introduced to this exercise. I'm fine with that, but I did open up https://github.com/exercism/problem-specifications/pull/1758 to possibly annotate error test cases with a specific scenario for easy filtering.

Reply:
I expected to see input validation or error scenario in plenty other test cases. My suggestion would be: YES, add it.